src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -7382,6 +7382,7 @@ def test_create_image_version_with_allow_replicated_location_deletion(self, reso
             self.check('safetyProfile.allowDeletionOfReplicatedLocations', False)
         ])
 
+    @unittest.skip("The value of Edge Zone name 'attdallas1' and 'microsoftlosangeles1' are invalid.")
```

Comment:
Could we consider updating the tested edge zone to the valid value

Reply:
It is caused by a breaking change from service side. We need to contact service team if we want to update it.
I think it is a little time consuming, so maybe we can merge it firstly and contact them to update it later.

Reply:
OK, sounds good

Reply:
Asked for the available values in https://github.com/Azure/azure-cli/issues/23745#issuecomment-2586522570